koschei/db.py

```diff
@@ -110,6 +110,15 @@ def bulk_insert(self, objects):
                 obj.id = obj_id
             self.expire_all()
 
+    def commit_no_expire(self):
```

Comment:
There is a mismatch between commit message and this function name (typo in commit msg?)